src/app/components/chat/window/window.vue

```diff
@@ -100,8 +103,13 @@
 									:collapse-height="60"
 									@require-change="isDescriptionCollapsed = $event"
 								>
-									<div class="chat-window-header-room-description chat-compiled-room-description">
-										<div class="anim-fade-in no-animate-xs" v-html="room.description"></div>
+									<div
+										class="chat-window-header-room-description chat-compiled-room-description"
+									>
+										<div
+											class="anim-fade-in no-animate-xs"
+											v-html="room.description"
+										></div>
```

Comment:
We could probably remove this altogether as there are descriptions in PM rooms.

Reply:
Yeah, it's currently unused. There is a task to remove room descriptions, which we will do later.
Since it will never be populated right now, we can keep it like this and remove it with that task.

(Also we can't use <translate> for that case, everything between translate tags has to be static)

Reply:
I did add TODO comments above both cases so we can easily find these once it's time to remove.

Reply:
I'll go ahead and resolve this then